priv/repo/migrations/20180523195208_create_daily_geo_metros.exs

```diff
@@ -0,0 +1,57 @@
+defmodule Castle.Repo.Migrations.CreateDailyGeoMetros do
+  use Ecto.Migration
+
+  def change do
+    create table(:daily_geo_metros, primary_key: false) do
+      add :podcast_id, :integer, null: false
+      add :episode_id, :uuid, null: false
```

Comment:
ah I see now. `episode_id` is a uuid not an integer.

Reply:
Yeah, i've sort of been going back and forth on `episode_guid` vs `episode_id`.  I'm (gradually) moving towards referring to it as `id` always, since that's how Feeder presents it ("guid" actually means something feed-specific to Feeder) http://feeder.prx.org/api/v1/episodes